options/options.js

```diff
@@ -19,6 +19,22 @@ function restoreOptions() {
   var value = '';
   var getting = '';
 
+  // set event click for "move-up" and "move-down" and set the text of those buttons
+  document.querySelectorAll('button[id]').forEach(function(item) {
+    if (item.id.endsWith("move-up")) {
+      document.getElementById(item.id).addEventListener('click', function () {
+        moveUp(item.id);
+      });
+      document.getElementById(item.id).innerHTML="▲";
+    }
+    else if(item.id.endsWith("move-down")) {
+      document.getElementById(item.id).addEventListener('click', function () {
+        moveDown(item.id);
+      });
+      document.getElementById(item.id).innerHTML="▼";
+    }
+  });
```

Comment:
```  
document.querySelectorAll('button[id]').forEach(function(item) {
document.getElementById(item.id).addEventListener('click', function () {
    if (this.endsWith("move-up")) {
        moveUp(item.id);
      this.innerHTML="▲";
    } else if(this.endsWith("move-down")) {
        moveDown(item.id);
      this.innerHTML="▼";
    }
  });
});
```

Reply:
less core more easy to mantain :-)

Reply:
Absolutely ok 👍 I'll change it as soon as

Reply:
This doesn't work